apps/communications/contacts/js/activities.js

```diff
@@ -96,12 +96,9 @@ var ActivityHandler = {
         break;
       case 'open':
         if (this.isvCardActivity(activity)) {
-          var self = this;
-          var dependency =
-                         '/shared/js/contacts/import/utilities/vcard_reader.js';
-          LazyLoader.load(dependency, function() {
-            self.getvCardReader(activity.source.data.blob,
-                                           self.readvCard.bind(self, activity));
+          LazyLoader.load('/contacts/js/activities_vcard.js', () => {
+            ActivityHandler.vCardActivityHandler.handle(activity,
```

Comment:
I think vcardActivityHandler does not need to be a property of ActivityHandler. It  can live in the window object as well